homeassistant/util/ssl.py

```diff
@@ -139,6 +147,14 @@ def client_context(
     return _SSL_CONTEXTS.get(ssl_cipher_list, _DEFAULT_SSL_CONTEXT)
 
 
+def create_client_context(
+    ssl_cipher_list: SSLCipherList = SSLCipherList.PYTHON_DEFAULT,
+) -> ssl.SSLContext:
+    """Return an independent SSL context for making requests."""
+    # This explicitly uses the non-cached version to create a client context
+    return _create_client_context(ssl_cipher_list)
```

Comment:
Looks like we are missing coverage for the new function

Reply:
Thank you for the review. I have added tests in an additional commit.